projects/github_scarecrow/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
is "scarecrow" a common term for this type of stuff? I've never heard of it before.
Maybe we can name it "github_filing_test" to be more descriptive?

Reply:
I guess you can leave the name as is because your repo is named scarecrow already.